book/06-github/sections/1-setting-up-account.asc

```diff
@@ -38,7 +38,7 @@ Then select the ``SSH keys'' section along the left-hand side.
 .The ``SSH keys'' link.
 image::images/ssh-keys.png[The ``SSH keys'' link.]
 
-From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key''.
+From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key.''
```

Comment:
Yeah, let's do that. If we leave it the way it is or change it the way you've got it, there will always be a subgroup who reads it and thinks "that's not how I'd do it."